app/analysis/edit/template.hbs

```diff
@@ -82,15 +82,18 @@
                   {{/sy-checkbox}}
                 {{/f.input}}
 
-                {{#if (or (eq reviewer session.data.user.id) session.data.user.isSuperuser)}}
-                  {{#f.input data-test-verified name='verified' as |fi|}}
-                    {{#sy-checkbox checked=fi.value on-change=fi.update}}
-                      Verified
-                      {{#if (eq model.verified null)}}{{not-identical-warning}}{{/if}}
-                      {{#unless (eq f.model.verified model.verified)}}{{changed-warning}}{{/unless}}
-                    {{/sy-checkbox}}
-                  {{/f.input}}
-                {{/if}}
+                {{#f.input data-test-verified name='verified' as |fi|}}
+                  {{#sy-checkbox
+                    checked=fi.value
+                    on-change=fi.update
+                    title=(unless (or (eq reviewer session.data.user.id) session.data.user.isSuperuser) 'Please set reviewer filter to yourself to verify reports.')
```

Comment:
Maybe better: "Please select yourself as "reviewer" to verify reports.

Reply:
Also, it might be worth assigning `(unless (or (eq reviewer session.data.user.id) session.data.user.isSuperuser)` to a variable like `canVerify`.